### DISCUSSION TITLE
Is the Hpool pool safe

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#4178] [General]
Hi,
Transactions have started in the Hpool pool, but the wallet wants my words, do you think it's safe?

**Comment by grandfatheryun** [comment]
> Hi,
> 
> Transactions have started in the Hpool pool, but the wallet wants my words, do you think it's safe? 

as your think，it is really not safe，please do not give your words to anybody

**Reply by unitedchia** [reply]
The current pools are really not safe. They require you to enter your private key and your mnemonic seed 24 word phrase. 
This is basically like posting a picture of your creditcard on facebook. Please do not do that.

There are a variety of pools preparing to launch with the official pooling protocol, which will be released on 05/17/2021 (see  [tweet of chia founder Bram Cohen).](https://twitter.com/bramcohen/status/1390407484179763202)

One solution is our pool [chiaunited.com](https://chiaunited.com). Feel free sign up and subscribe for updates. The pool will be launched soon. 

Stay Tuned!

**Reply by oytp** [reply]
Yep. But when other pools start, the income will be really low.

**Comment by GeorgePapageorgakis** [comment]
You shouldnt join hpool if you dont want to see chia coins going to zero value. They already more than 50% of the network. which means that they can allocate block veification on their wallets only. This is getting out of hand.
